Assets/Python/Victory.py

```diff
@@ -950,9 +950,8 @@ def checkTurn(iGameTurn, iPlayer):
 				lose(iRussia, 0)
 				
 		if isPossible(iRussia, 0):
-			if teamRussia.isHasTech(iRailroad):
-				if isConnectedByRailroad(iRussia, Areas.getCapital(iRussia), lSiberianCoast):
-					win(iRussia, 0)
+			if isConnectedByRailroad(iRussia, Areas.getCapital(iRussia), lSiberianCoast):

```

Comment:
Please keep this, as I recently commented in the forum that check is in to avoid unnecessary computation. Apparently it is just missing for the UHV display.